packages/block-library/src/buttons/editor.scss

```diff
@@ -1,8 +1,14 @@
-.wp-block-buttons .wp-block.block-editor-block-list__block[data-type="core/button"] {
+.wp-block-buttons
+.wp-block.block-editor-block-list__block[data-type="core/button"] {
 	display: inline-block;
 	width: auto;
 }
 
+.wp-block-buttons.is-vertical
+.wp-block.block-editor-block-list__block[data-type="core/button"] {
```

Comment:
This also shouldn't be two separate lines.

Reply:
Fixed.